### PR TITLE
Update CHANGELOG for 0.10.2 release

### PR DESCRIPTION
**Pull request by flarnie**
This is going to be an extra awesome release - thanks @sophiebits for fixing that "This Selection object doesn't have any Ranges" bug, and @jessebeach for all the a11y improvements!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,29 @@ Notable changes to Draft.js will be documented in this file.
...
 Changes to `src` are live in production on facebook.com at the time of release.
 
+## 0.10.2
+
+### Added
+* Change aria-owns to aria-controls in draft.js ([@jessebeach](https://github.com/jessebeach) in [7f0cab28](https://github.com/facebook/draft-js/commit/7f0cab28386ec4bde8ec6289377bff9e53cd019b))
+  * Adds support for `ariaControls` prop
+
+### Changed
+* Change aria-owns to aria-controls in draft.js ([@jessebeach](https://github.com/jessebeach) in [7f0cab28](https://github.com/facebook/draft-js/commit/7f0cab28386ec4bde8ec6289377bff9e53cd019b))
```
Comment (sophiebits) [inline CHANGELOG.md]: Dupe line?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,29 @@ Notable changes to Draft.js will be documented in this file.
...
+  * Deprecates support of `ariaControls` and `ariaOwneeID` props
+* Deprecate use of `ariaHasPopup` prop in draft.js ([@jessebeach](https://github.com/jessebeach) in [744e9b4e](https://github.com/facebook/draft-js/commit/744e9b4eb4810797a93c66591fea6f9cac533b4b))
```
Comment (sophiebits) [inline CHANGELOG.md]: Maybe say:

> `ariaExpanded` should be used instead if an input is showing a dropdown with options.

File: CHANGELOG.md (hunk 1 above)
Comment (flarnie) [reply CHANGELOG.md]: I thought it qualified as a 'changed' as well as an 'added' item, but probably could just list it under one or the other.

Comment (sophiebits) [reply CHANGELOG.md]: I think it is OK to have under both if the wording is different but it's confusing if it's identical.

**Comment by flarnie** [comment]
Accidentally pushed the branch to the Facebook repo instead of my fork - closing this in favor of making a PR from my repo.

**Pull request closed without merging**